js/render-advertisements.js

```diff
@@ -0,0 +1,97 @@
+import { generateData } from './mock-data.js';
+import { hideElement } from './util.js';
+
+const QuerySelectors = {
```

Comment:
Используется как словарь, множественное число не нужно
```suggestion
const QuerySelector = {
```